docs/en/contributor/Contributor-Getting-Started.md

```diff
@@ -294,6 +294,28 @@ Congratulations! You have successfully contributed to Alluxio! Thank you for joi
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/QsbM804rc6Y" frameborder="0" allowfullscreen></iframe>
 
+## Submit Your Feature
+
+If you have a brilliant idea about a new Alluxio feature, we highly encourage you to implement it and 
+contribute to Alluxio repository.
+
+1. Create a Github issue in [Alluxio repository](https://github.com/Alluxio/alluxio/issues)
+regarding your feature.
+1. Attach a feature design document following the [template]({{ '/templates/design-doc-template.docx' | relativize_url }})
+in your Github issue. The design document should follow the template and provide information regarding each section.
+We recommend using a public Google doc for more efficient collaboration and discussions.
+1. Tag your issue `type-feature`. Alluxio members periodically check all open issues and will allocate reviewers to the design document.
+1. The reviewers (assignees on the corresponding Github issue) will review the feature design document and provide feedback or questions.
+There can be technical discussions with or among the reviewers, or other active Alluxio community members.
+Feel free to raise questions and seek help from the reviewers and community members, if you are uncertain about specific 
+design decisions. We highly recommend listing options for the feature and corresponding pros and cons.
```

Comment:
`We highly recommend listing options for the feature and corresponding pros and cons`

This sounds like something that should be _required_ in the design document template?

Reply:
`We highly recommend listing options` -> `Please also list options`